awx/main/conf.py

```diff
@@ -80,11 +80,11 @@
 )
 
 register(
-    'PROXY_IP_WHITELIST',
+    'PROXY_IP_ALLOWED_LIST',
     field_class=fields.StringListField,
-    label=_('Proxy IP Whitelist'),
+    label=_('Proxy IP Allowed List'),
     help_text=_("If Tower is behind a reverse proxy/load balancer, use this setting "
-                "to whitelist the proxy IP addresses from which Tower should trust "
+                "to allow the proxy IP addresses from which Tower should trust "
```

Comment:
This construction, "to allow ... from which", seems particularly awkward to me.  I think maybe "to configure" works better here.